### DISCUSSION TITLE
Galaxy Brain Pls

### DISCUSSION THREAD

**Discussion by chaymankala** [drknzz/GitHub-Achievements#382] [Q&A] [answered]
pleasee

**Comment by chaymankala** [comment, marked as answer]
Is this?

**Reply by SatyamPote** [reply]
ok

**Comment by Coolbreeze151** [comment]
To get the Galaxy Brain achievement you need to answer at least 2 discussions and have the OP accept your answers.

If you would like, we can exchange with each other.

https://github.com/Schweinepriester/github-profile-achievements/discussions/945#discussioncomment-10782175

https://github.com/Schweinepriester/github-profile-achievements/discussions/946#discussioncomment-10782183

**Comment by SatyamPote** [comment]
https://github.com/SatyamPote/CodeStacks

Help each other to get more stars

**Comment by byKosta** [comment]
I'm also
